src/components/likedByContainer/LikedByContainer.tsx

```diff
@@ -1,40 +1,64 @@
-import {
-    SAMPLE1_LIKED_BY_PROFILE_PICTURE,
-    SAMPLE2_LIKED_BY_PROFILE_PICTURE,
-} from '../../constants';
+import { MOCK_DATA } from '../../mockText';
+import { AND, LIKED_BY, OTHER, OTHERS, PROFILE_PICTURE_ALT } from '../../constants';
+import { st, classes } from './LikedByContainer.st.css';
+import { classes as generalClasses } from '../../style/general.st.css';
 import type { User } from '../../types';
 
 interface ILikedByContainer {
     allUsersWhoLiked: User[];
 }
 
 const LikedByContainer = ({ allUsersWhoLiked }: ILikedByContainer) => (
-    <div>
-        <div>
-            <div>
-                <img src={SAMPLE1_LIKED_BY_PROFILE_PICTURE} alt="profile Picture"></img>
+    <div className={classes.root}>
+        <div className={classes.likeByWho}>
+            <div className={classes.UserLikeByPic}>
+                {allUsersWhoLiked.map((user, index) => {
+                    return index > 2 || index === 0 ? (
+                        <></>
+                    ) : (
+                        <img
+                            className={st(
+                                classes.likeProfilePicture,
+                                classes.otherUserLikeByPic,
+                                generalClasses.profilePicture
+                            )}
+                            src={user.profilePicture}
+                            alt={PROFILE_PICTURE_ALT}
+                        ></img>
+                    );
+                })}
                 <img
-                    src={SAMPLE2_LIKED_BY_PROFILE_PICTURE}
-                    alt="profile Picture"
-                    id="lastUserLikeByPic"
+                    className={st(classes.likeProfilePicture, generalClasses.profilePicture)}
+                    src={allUsersWhoLiked[0].profilePicture}
+                    alt={PROFILE_PICTURE_ALT}
                 ></img>
             </div>
-            <div>
-                <span>Liked by</span>
-                <span>
-                    {allUsersWhoLiked.length > 1 ? (
+            <div className={classes.likeContentContainer}>
+                <span>{LIKED_BY}</span>
+                <span className={st(generalClasses.boldText, generalClasses.text)}>
+                    {allUsersWhoLiked.length === 2 ? (
+                        <>
+                            {allUsersWhoLiked[1].userName}
+                            <span className={generalClasses.text}>{AND}</span>
+                            <span className={st(generalClasses.boldText, generalClasses.text)}>
+                                {allUsersWhoLiked.length - 1} {OTHER}
+                            </span>
+                        </>
```

Comment:
not sure i see the difference between these two cases.

why not just use ``allUsersWhoLiked.length > 1``

Reply:
try to change to condition